DataTool/ToolLogic/List/Misc/ListWorkshop.cs

```diff
@@ -212,6 +226,17 @@ public class WorkshopDefinition {
             public IEnumerable<WorkshopParameter> Parameters;
         }
 
+        public class ExtensionVal {
+            public teResourceGUID DisplayName_Id;
```

Comment:
We don't need the id of the display name

Reply:
k will remove it